packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -3,11 +3,30 @@
 
 import { E } from '@endo/eventual-send';
 import { AmountMath } from '@agoric/ertp';
-import { offerTo } from '@agoric/zoe/src/contractSupport/index.js';
+import {
+  floorMultiplyBy,
+  offerTo,
+} from '@agoric/zoe/src/contractSupport/index.js';
 import { makeTracer } from '../makeTracer.js';
 
 const trace = makeTracer('LIQ');
 
+/**
+ * @param {Amount<'nat'>} proceeds
+ * @param {Amount<'nat'>} debt - after incurring penalty
+ * @param {Amount<'nat'>} penaltyPortion
+ */
+const partitionProceeds = (proceeds, debt, penaltyPortion) => {
+  const isShortfall = !AmountMath.isGTE(proceeds, debt);
+  const debtPaid = isShortfall ? proceeds : debt;
```

Comment:
```suggestion
  const debtPaid = AmountMath.min(proceeds, debt);
```

Reply:
👍 I suppose `isShortfall` was defined for tracing but I already took it out of the trace because `debtPaid` implies it.